doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,70 @@
+
+# PaddlePaddle Client
+PaddlePaddle clinet is a command line tool, before startting a cluster train, you need to install it on your latop, you can sumit a cluster train job looks like:
+
+```bash
+paddle k8s job <job-name>
+    --package-path= ./demo
+    --module=demo.train
+    --pserver-count=2
+    --trainer-count=4
+    --output=${OUTPUT_DIR}
+    --input=${INPUT_DIR}
+```
+
+- job-name: you can specify a name for every job, and the name should be uniq.
+- package-path: your package files, including py file and the dependencies.
+- module: Include main function, it is the trainer entrance.
+- pserver-count: specify the parameter process count.
+- trainer-count: specify the trainer process count.
+- output: output path on distributed storage.
+- input: input path on distributed storage.
+
+For this command, it will setup a master process, 2 pserver processes and 4 triner processes on cluster.
+
+# Master process
+- Setup master process
+
+  While user submit a distributed train job through PaddlePaddle client, it will setup a master process on cluster, the master process provids a http service for receiving package files and some cluster parameter.
```

Comment:
For the import consideration, for each job, they are independent. Master is responsible for setup, stop, report status for the job, so maybe one master process for each job is a simple plan, it does not care about sth. such as user access etc.. 

On the other hand, pre start multiple master process maybe a better plan, user does need to setup a new master process when submit a job.  If you have some idea, let's discuss this one.